source/includes/administration/_pricings.md

```diff
@@ -188,6 +215,26 @@ curl "https://cloudmc_endpoint/rest/pricings/03bc22bd-adc4-46b8-988d-afddc24c0cb
             "en": "sfg",
             "fr": "sdfg"
           },
+          "pricingTiers": [
+            {
+              "pricingMode": "FLAT_FEE",
+              "upperBound": 1000.5,
```

Comment:
If possible I would have the bound value right next to each other with the lower bound on top

Reply:
2e481388659b6fc437a11c7e6b96555b1d219ffd